docs/source/usage/workflows/memoryPerDevice.py

```diff
@@ -27,8 +27,9 @@
 
 cell_size = 0.8e-6 / 384.  # 2.083e-9 m
 
-y0 = 0.5e-6  # position of foil surface (m)
-y1 = 1.0e-6  # target thickness (m)
+y0 = 0.5e-6  # position of foil front surface (m)
+delta_y = 1.0e-6  # target thickness (m)
```

Comment:
@pordyna Would you add the suggestions from @n01r ?

Reply:
@pordyna ping :) whenever you have the time, please feel free to update the pull request by pushing the suggested changes to its branch :)